curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-provide-explanations-when-helping-others/6626273eb00c031f4988f129.md

```diff
@@ -0,0 +1,48 @@
+---
+id: 6626273eb00c031f4988f129
+title: Task 3
+challengeType: 19
+dashedName: task-3
+---
+
+# --description--
+
+Frustration is a feeling of being upset or annoyed because of inability to change or achieve something. For example, it can be frustrating when you're trying to learn a new skill but find it difficult. This emotion often comes when something is hard or doesn't go as planned.
```

Comment:
```suggestion
`Frustration` is a feeling of being upset or annoyed because of inability to change or achieve something. For example, it can be frustrating when you're trying to learn a new skill but find it difficult. This emotion often comes when something is hard or doesn't go as planned.
```

Are we teaching them the word here?

Reply:
I asked @nieldakarla if I should teach them "frustration", "frustrated" and "frustrating" and "frustrate" when creating this PR. She said no need as the question didn't mention these variations. Now I'm not sure - maybe teach them "frustration"?